apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/style.scss

```diff
@@ -52,26 +55,15 @@
 			.block-editor-block-contextual-toolbar {
 				display: none;
 			}
-		}
 
-		&.block-editor-block-list__block {
-			// Need to get super specific to override the core css selectors:
-			&,
-			&.has-child-selected,
-			&.is-navigate-mode,
-			&.is-hovered {
-				> .block-editor-block-list__block-edit {
-					&::before {
-						transition: none;
-						border: none;
-						outline: none;
-						box-shadow: none;
-					}
-					> .block-editor-block-list__breadcrumb {
-						display: none;
-					}
-				}
+			& > .editor-block-list__block-edit.block-editor-block-list__block-edit::before {
+				background-color: rgba( #000000, 0.5 );
+				z-index: 1;
 			}
 		}
+
+		&:not( .is-selected ):hover {
+			outline: #e2e4e7 solid;
```

Comment:
For follow up outside of this PR I wonder if it'd be good to start a named colors file.

Reply:
I definitely agree, especially as a way of consolidating a lot of the styling hacks we're doing to gutenify this.